unity/Assets/Scripts/Content/DictionaryI18n.cs

```diff
@@ -398,16 +398,40 @@ public string GetValue(string key)
             return rawData;
         }
 
+        /// <summary>
+        /// Escape entry values
+        /// </summary>
+        /// <param name="entry">entry value to replace</param>
+        /// <returns>Escaped entry, with newlines and quotes handled</returns>
+        protected string EscapeEntry(string entry)
+        {
+            if (entry == null) throw new ArgumentNullException("entry");
+            // Handle enter
+            string escapedEntry = entry.Replace("\r\n", "\n");
+            escapedEntry = escapedEntry.Replace("\r", "\n");
+            escapedEntry = escapedEntry.Replace("\n", "\\n");
+            if (escapedEntry.Length > 2 && escapedEntry.StartsWith("\"") && escapedEntry.EndsWith("\""))
```

Comment:
I'm not convinced by this solution.  Currently this will only have quotes at the start and end if the user specifically added them, this should probably not be removed.  Also the escaping and external quotes should probably happen on any text coming into this function.

Reply:
Ok, I will try to change it that way, that there are always quotes around the escaped text.

Reply:
Made the changes